migrations/0003_convert_unix_timestamp_to_date.sql

```diff
@@ -0,0 +1,4 @@
+-- Migration number: 0003 	 2025-01-29T16:47:52.305Z
+
+-- Convert the birth_date column from a Unix timestamp to a date in YYYY-MM-DD format.
+UPDATE birthdays SET birth_date = DATE(birth_date, 'unixepoch') WHERE DATE(birth_date) IS NOT birth_date;
```

Comment:
If we want to be really sure that our data stays clean, a CHECK constraint can be added to the SQLite database like this: https://sqlite.org/forum/info/a1414bf570006db8

However that requires re-creating the table since a CHECK constraint cannot be added after creation. Which would make the migration more complicated so I skipped it for now. :D